ui/component/comment/view.jsx

```diff
@@ -96,6 +98,7 @@ function Comment(props: Props) {
   const dislikesCount = (othersReacts && othersReacts.dislike) || 0;
   const totalLikesAndDislikes = likesCount + dislikesCount;
   const slimedToDeath = totalLikesAndDislikes >= 5 && dislikesCount / totalLikesAndDislikes > 0.8;
+  const commentByOwnerOfContent = creatorUri === authorUri;
```

Comment:
Instead of passing `creatorUri` all the way down from `page/show` you can use `makeSelectClaimForUri` in `comment/index.js` ([example](https://github.com/lbryio/lbry-desktop/blob/master/ui/component/fileActions/index.js#L19))

Then you can do
```
const commentByOwnerOfContent = claim && claim.signing_channel && claim.signing_channel.permanent_url === authorUri
```